### PR TITLE
py-pyfaidx: new package at 0.5.5.2

### PR DESCRIPTION
**Pull request by molecuul**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pyfaidx/package.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyfaidx(PythonPackage):
+    """pyfaidx: efficient pythonic random access to fasta subsequences"""
+
+    homepage = "http://mattshirley.com/about"
```
Comment (healther) [inline var/spack/repos/builtin/packages/py-pyfaidx/package.py]: https://pypi.org/project/pyfaidx/

File: var/spack/repos/builtin/packages/py-pyfaidx/package.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    url      = "https://files.pythonhosted.org/packages/75/a5/7e2569527b3849ea28d79b4f70d7cf46a47d36459bc59e0efa4e10e8c8b2/pyfaidx-0.5.5.2.tar.gz"
```
Comment (healther) [inline var/spack/repos/builtin/packages/py-pyfaidx/package.py]: https://pypi.io/packages/source/p/pyfaidx/pyfaidx-0.5.5.2.tar.gz

Comment (molecuul) [reply var/spack/repos/builtin/packages/py-pyfaidx/package.py]: fixed; just wondering, where did you find the link?

File: var/spack/repos/builtin/packages/py-pyfaidx/package.py (hunk 1 above)
Comment (molecuul) [reply var/spack/repos/builtin/packages/py-pyfaidx/package.py]: fixed

**Pull request merged**